x-pack/plugins/canvas/public/components/embeddable_flyout/flyout.tsx

```diff
@@ -67,7 +67,7 @@ export const AddEmbeddablePanel: React.FunctionComponent<FlyoutProps> = ({
   availableEmbeddables,
   ...restProps
 }) => {
-  const labsService = useLabsService();
+  const labsService = useMemo(() => presentationUtilService.labsService, []);
```

Comment:
Is a memozied version of labsService even needed. The variable is only used once, why not just directly use `presentationUtilService.labsService` instead of creating this constant.